tests/everest/test_everlint.py

```diff
@@ -590,7 +590,7 @@ def test_lint_everest_models_jobs():
 def test_overloading_everest_models_names():
     config = yaml_file_to_substituted_config_dict(SNAKE_OIL_CONFIG)
     for job in collect_forward_models():
```

Comment:
Is it still valid to test the `collect_forward_models()` if they now go through the plugin system instead?